config.map

```diff
@@ -2,7 +2,7 @@ SCRAM_ARCH=slc7_amd64_gcc900;PKGTOOLS_TAG=V00-33-XX;CMSDIST_TAG=IB/CMSSW_11_3_X/
 SCRAM_ARCH=slc7_amd64_gcc900;PKGTOOLS_TAG=V00-33-XX;CMSDIST_TAG=IB/CMSSW_11_3_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_11_3_DBG_X;DOCKER_IMG=cmssw/cc7:amd64;BUILD_HOUR=23;BUILD_DAY=4;
 SCRAM_ARCH=slc7_amd64_gcc900;PKGTOOLS_TAG=V00-33-XX;CMSDIST_TAG=IB/CMSSW_11_3_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_11_3_SKYLAKEAVX512_X;DOCKER_IMG=cmssw/cc7:amd64;BUILD_HOUR=23;BUILD_DAY=4;
 SCRAM_ARCH=slc7_amd64_gcc900;PKGTOOLS_TAG=V00-33-XX;CMSDIST_TAG=IB/CMSSW_11_3_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_11_3_GPU_X;DOCKER_IMG=cmssw/cc7:amd64;BUILD_HOUR=23;
-SCRAM_ARCH=slc7_amd64_gcc900;PKGTOOLS_TAG=V00-33-XX;CMSDIST_TAG=IB/CMSSW_11_3_X/clang;RELEASE_BRANCH=CMSSW_11_3_CLANG_X;RELEASE_QUEUE=CMSSW_11_3_CLANG_X;DOCKER_IMG=cmssw/cc7:amd64;BUILD_HOUR=23;
+SCRAM_ARCH=slc7_amd64_gcc900;PKGTOOLS_TAG=V00-33-XX;CMSDIST_TAG=IB/CMSSW_11_3_X/master;RELEASE_BRANCH=CMSSW_11_3_CLANG_X;RELEASE_QUEUE=CMSSW_11_3_CLANG_X;DOCKER_IMG=cmssw/cc7:amd64;BUILD_HOUR=23;
```

Comment:
I think we can also change the following as there are no special clang related changes in `CMSSW_11_3_CLANG_X` branch
```
RELEASE_BRANCH=CMSSW_11_3_CLANG_X
```
to
```
RELEASE_BRANCH=CMSSW_11_3_X
```

Reply:
looks like it.
https://github.com/cms-sw/cmssw/compare/CMSSW_11_3_CLANG_X...CMSSW_11_3_X
I can't erase branches from cmssw you have to do it